### ISSUE TITLE
Issue with Training Time and GPU Usage on 1080Ti

### ISSUE THREAD

**Issue by hmc-cs-jraygoza** [matterport/Mask_RCNN#1742]
Hi all, I'm trying to re-train the Mask RCNN (starting with the COCO weights). My goal is for the model to recognize two new classes. The instances of one class are rectangular but have rounded edges, while the instances of the other class are also rectangular but have sharp edges. All images of the rounded-edge class have the dimensions 200 x 279, and all images of the sharp-edge class have the dimensions 200 x 292. Per class, my annotations are identical for all of the images containing an instance of that class because the images of that class all have the same dimensions and because an instance always takes up the entire image (with the exception of the edges for instances of the rounded-edge class).  

I have 2000 images per class, but I will need to train using less images because I only have one 1080Ti (11GB) and an i7-8700K. I'm on Windows 10 and using `tensorflow-gpu = 1.14.0`, `Keras = 2.2.5`, and `CUDA = 10.0`. I have not been able to make it past epoch 1/30 when training, _even after running for two hours_. I've noticed that my GPU RAM fills up to 9.4GB consistently and my CPU usage spikes, but my GPU usage does not spike and instead stays under 2% or so. Though I've changed the configurations frequently, this is what I currently have:
``` 
IMAGES_PER_GPU = 2

NUM_CLASSES = 1 + 2 
 
IMAGE_MIN_DIM = 256

IMAGE_MAX_DIM = 320

TRAIN_ROIS_PER_IMAGE = 4

RPN_ANCHOR_SCALES = (256, 320)

STEPS_PER_EPOCH = 100

VALIDATION_STEPS = 5

DETECTION_MIN_CONFIDENCE = 0.9
```
I'm not attached to these configurations in any way, as I have just been experimenting in hopes of reducing training time.

Does anybody know if I should be witnessing a spike in GPU usage while training, rather than a spike in only GPU RAM? If so, does the problem seem to be my configurations, library issues, or am I just hitting the limits of my system? One final thing to note is that I do have a few `Tensorflow` and `numpy` deprecation warnings, seeing as this repo needs to have a few function calls updated. Any guidance would be deeply appreciated.

**Comment by hmc-cs-jraygoza** [comment]
UPDATE: This issue has been resolved. My annotations were being improperly loaded in the `load_mask` function, which resulted in the training never progressing. GPU usage now consistently spikes up to ~13% while training.

**Comment by AliaChen** [comment]
Hi, I used the GTX2080Ti graphics card, but the training speed was hugly slow, and it would appear that   "UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.Converting sparse IndexedSlices to a dense Tensor of unknown shape".  Waiting for your Enthusiastic guidance.

**Comment by hmc-cs-jraygoza** [comment]
I get the same warning so that may not be the issue. What are the last few lines output to the terminal when you run one of the scripts in the `samples` directory? Also, in whatever script you are running, what functions have been modified?
